sdk/eventhub/Azure.Messaging.EventHubs.CheckpointStore.Blob/Directory.Build.props

```diff
@@ -28,6 +28,6 @@
     <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
 
     <!-- If there was no override specified, set the type of reference to use for client libraries from the Azure SDK repository -->
-    <UseProjectReferenceToAzureClients Condition="'$(UseProjectReferenceToAzureClients)' == ''">$(UseProjectReferenceToAzureCore)</UseProjectReferenceToAzureClients>
+    <UseProjectReferenceToAzureClients Condition="'$(UseProjectReferenceToAzureClients)' == ''">true</UseProjectReferenceToAzureClients>
```

Comment:
@jsquire I changed this value to `true` because correct packages are not out yet.

Reply:
All good;  We'll be flipping things once we release Event Hubs and storage goes as well.   Appreciate the heads-up.

//cc: @kinelski